### PR TITLE
Added get and post request

### PR DESCRIPTION
**Pull request by chetak123**
## Related Issue

- Fixes # (issue) #40 

## Proposed Changes 

I have added get and post requests and fixed the db_function for "date_of_release", changed name of router to singer_router and rap_router.js

cc @yashgangwar7558 @AbhijithGanesh 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-var models = require("express-cassandra")
```
Comment (AbhijithGanesh) [inline src/index.js:1]: This is a mandatory change, please revert.

**Review by AbhijithGanesh — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (chetak123) [reply src/index.js:1]: ohh, actually on vscode it's showing not used anywhere in index.js so I think I deleted it by mistake

**Review by yashgangwar7558 — APPROVED** [review]
Great work, lgtm!

**Review by AbhijithGanesh — APPROVED** [review]
lgtm

**Pull request merged**